src/components/LeftSidebar/NewGroupConversation/SetContacts/ContactSelectionBubble/ContactSelectionBubble.vue

```diff
@@ -0,0 +1,106 @@
+<!--
+  - @copyright Copyright (c) 2020 Marco Ambrosini <marcoambrosini@pm.me>
+  -
+  - @author Marco Ambrosini <marcoambrosini@pm.me>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+	<div class="contact-selection-bubble">
+		<AvatarWrapperSmall
+			:id="participant.id"
+			class="contact-selection-bubble__avatar"
+			:name="participant.label"
+			:source="participant.source"
+			:disable-menu="true"
+			:disable-tooltip="true" />
+		<span class="contact-selection-bubble__username">
+			{{ trimmedName }}
+		</span>
+		<button
+			class="icon-close contact-selection-bubble__remove"
+			@click="removeParticipantFromSelection(participant)" />
+	</div>
+</template>
+
+<script>
+import AvatarWrapperSmall from '../../../../AvatarWrapperSmall'
+export default {
+	name: 'ContactSelectionBubble',
+
+	components: {
+		AvatarWrapperSmall,
+	},
+
+	props: {
+		participant: {
+			type: Object,
```

Comment:
Can't we unpack this os it's more clear which properties are being used/required?

Reply:
Makes sense, I'll look into it

Reply:
Nope, I need the whole participant object for comparison purpose in the store